dds/DCPS/DataReaderImpl.cpp

```diff
@@ -1147,11 +1147,10 @@ DataReaderImpl::get_matched_publication_data(
 DDS::ReturnCode_t
 DataReaderImpl::enable()
 {
-  //According spec:
-  // - Calling enable on an already enabled Entity returns OK and has no
-  // effect.
+  // According to spec:
+  // - Calling enable on an already enabled Entity has no effect and returns OK.
   // - Calling enable on an Entity whose factory is not enabled will fail
-  // and return PRECONDITION_NOT_MET.
+  //   and return PRECONDITION_NOT_MET.
```

Comment:
Why are these lines changed in this PR?  Doesn't seem related to this task.

Reply:
Just a little typo and format correction for better understanding of the code.